src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {Resizable} from 're-resizable';
+import classnames from 'clsx';
+import './ResizableBox.scss';
+import HandleResize from '~/icons/HandleResize';
+import {zones, EnableZones, ResizableBoxProps} from './ResizableBox.types';
+
+export const ResizableBox: React.FC<ResizableBoxProps> = ({
+    enable = ['right'],
+    minWidth = 50,
+    maxWidth = 200,
+    defaultSize = {width: '100%', height: 'auto'},
+    className = '',
+    size,
+    children = null,
+    onResizeStart,
+    onResizing,
+    onResizeStop,
+    ...props
+}) => {
+    const enableZones: EnableZones = {};
+
+    zones.forEach(zone => {
+        enableZones[zone] = enable.indexOf(zone as any) > -1;
+    });
+
+    return (
+        <Resizable
+            data-role="resizable-panel"
```

Comment:
Adding the `role` type  definition on `ResizableBoxProps` seems fix the typescript error. In that way, we avoid some breaking changes for our selenium tests.

Reply:
Ahh, nice. I'm not sure why that works but it does! I've made the update 😄